roles/dtc/common/templates/ndfc_create_fabric.j2

```diff
@@ -138,6 +138,7 @@
 {% if (underlay.isis.authentication_enable | default(defaults.vxlan.underlay.isis.authentication_enable) | title) == 'True' %}
   ISIS_AUTH_KEYCHAIN_NAME: {{ underlay.isis.authentication_keychain_name | default(omit) }}
   ISIS_AUTH_KEYCHAIN_KEY_ID: {{ underlay.isis.authentication_key_id | default(defaults.vxlan.underlay.isis.authentication_key_id) }}
+  ISIS_AUTH_KEY: {{ underlay.isis.authentication_key }}
```

Comment:
Ok just to make sure I understand.  If `authentication_key` is not set in the data model then the rule below will catch it right?